### ISSUE TITLE
palette by default

### ISSUE THREAD

**Issue by stvv0000** [alexkay/spek#283]
i want to set by default green-blue palette, it possible to do that? when i close window palette is changes to purple-red

**Comment by Kartoffelbreili** [comment]
Yea the green blue palette is definitely easier to see

**Comment by Perplexer1** [comment]
YES to this and all other settings that should be saveable now that a lot of customization is possible in Spek.
